concourse/steps/release.mako

```diff
@@ -107,6 +109,13 @@ import github.util
 import gitutil
 
 import git
+try:
+  import magic
+except ImportError:
+  # hack: install libmagic as workaround until libmagic will be installed into job-image
+  # -> remove this again
```

Comment:
`libmagic` is installed into job-image by now -> https://github.com/gardener/cc-utils/commit/108702b5a0f7c9cedf09524bde863cd49a1aad0a

Reply:
good catch - removed